lms/static/styles/frontend_apps.scss

```diff
@@ -30,6 +30,7 @@
 
 // Shared styles from frontend-shared package
 @use '@hypothesis/frontend-shared/styles';
+@use '@hypothesis/frontend-shared/styles/util';
```

Comment:
Part of my thinks we should get all the exported classes in one go, this feels a bit confusing, but I realize it's still a new package.

Reply:
Danger zone! Recall from this PR yesterday: https://github.com/hypothesis/frontend-shared/pull/66 that all utility classes are about to get a prefix. Also, there is documentation here: https://github.com/hypothesis/frontend-shared/tree/main/styles about what the current package SASS API is, and (in the FAQ) section, specifically advises against using any utility classes directly. In short, this class won't exist after the next package release.